src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static deleteDeadAnimal(animal) {
+    this.alive = this.alive.filter(name => name !== animal);
+  }
+
+  constructor(name) {
+    this.health = 100;
+    this.name = name;
+
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
+  }
```

Comment:
How can this animal stop hiding? 🙂 Better to change `hidden` value to opposite here

Reply:
I was wrong, ignore this comment 🙃